statusHandler/termuic/termuiRenders/widgetsRender.go

```diff
@@ -184,14 +191,47 @@ func (wr *WidgetsRender) prepareChainInfo() {
 	wr.chainInfo.Rows = rows
 }
 
+func (wr *WidgetsRender) prepareBlockInfo() {
+	//7 rows and one column
+	numRows := 7
+	rows := make([][]string, numRows)
+
+	currentBlockHeight := wr.getFromCacheAsUint64(core.MetricSynchronizedRound)
+	//currentBlockHeight is equals with MetricSynchronizedRound + 1
+	currentBlockHeight++
+	rows[0] = []string{fmt.Sprintf("Current block height: %v", currentBlockHeight)}
+
+	numTransactionInBlock := wr.getFromCacheAsUint64(core.MetricNumTxInBlock)
+	rows[1] = []string{fmt.Sprintf("Nr Transactions in block: %v", numTransactionInBlock)}
```

Comment:
I do not like Nr part of the message. Maybe replace with "Num" . Also, Transactions, Miniblocks should start with lower case.

Reply:
done, now start with lower case letter.